runtime/rasdump/javadump.cpp

```diff
@@ -89,6 +111,42 @@ static jvmtiIterationControl regionIteratorCallback (J9JavaVM* vm, J9MM_IterateR
 static UDATA getObjectMonitorCount	(J9JavaVM *vm);
 static UDATA getAllocatedVMThreadCount (J9JavaVM *vm);
 
+
+/**
+ * This function is used to get the Redhat OS distribution details
+ * @param [out] OS Distribution Name
+ *
+ * @return returns 0 on success, any other -1 code on failure
+ */
+
```

Comment:
Please omit the blank line between the description and the forward declaration.
Also, please leave a blank line before the next desciption.
This should reduce confusion about how comments are paired with those declarations.